frontend/packages/kubevirt-plugin/integration-tests/tests/vm.guest.agent.scenario.ts

```diff
@@ -0,0 +1,227 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  click,
+  deleteResource,
+  waitForStringInElement,
+  getDropdownOptions,
+  selectDropdownOption,
+  selectDropdownOptionById,
+  createResource,
+} from '@console/shared/src/test-utils/utils';
+import {
+  consoleTypeSelector,
+  consoleTypeSelectorId,
+  rdpServiceNotConfiguredElem,
+  desktopClientTitle,
+  launchRemoteViewerButton,
+  launchRemoteDesktopButton,
+  manualConnectionTitle,
+  rdpManualConnectionTitles,
+  rdpManualConnectionValues,
+  networkSelectorId,
+} from '../views/consolesView';
+import * as dashboardView from '../views/dashboard.view';
+import * as vmView from '../views/virtualMachine.view';
+import * as disksView from '../views/vm.disks.view';
+import {
+  VM_CREATE_AND_EDIT_TIMEOUT_SECS,
+  PAGE_LOAD_TIMEOUT_SECS,
+  VM_CREATE_AND_EDIT_AND_CLOUDINIT_TIMEOUT_SECS,
+  VM_WITH_GA_CREATE_AND_EDIT_CLOUDINIT_TIMEOUT_SECS,
+  GUEST_AGENT_FIELD_TIMEOUT_SECS,
+} from './utils/constants/common';
+import { VirtualMachine } from './models/virtualMachine';
+import { getFakeWindowsVM } from './utils/templates/windowsVMForRDPL2';
+import { multusNAD } from './mocks/mocks';
+
+const VM_NAME = 'windows-rdp';
+const VM_IP = '123.123.123.123';
+
+describe('Tests involving guest agent', () => {
+  let vm: VirtualMachine;
+
+  beforeAll(async () => {
+    // Extending default timeout due to time required for guest agent to load
```

Comment:
ops, could you remove the comment please? :)

Reply:
Good catch! =) Fixed!